script/sync-ghes/settings.json

```diff
@@ -2,7 +2,6 @@
   "folders": [
     "../../ci",
     "../../automation",
-    "../../deployments",
```

Comment:
Do we need the `partners` array as we're removing this entry as that array might not get used?

Reply:
Yes, we need it. It is being used in the script.